.github/workflows/main.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
Is this change for the required minimum Python version or just a change for what Python versions is used in CI? If the former, there is a line in the README that should be updated: https://github.com/seekthermal/seekcamera-python/blob/main/README.md#installation-open_file_folder

Reply:
This was for the CI. For the next release, I'll update the readme though since we don't what to support 3.6 and 3.7 since they are both EOL.